foxglove/cmd/info.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/foxglove/foxglove-cli/foxglove/console"
+
+	"github.com/spf13/cobra"
+)
+
+func executeInfo(baseURL, clientID, token, userAgent string) error {
+	isUsingApiKey, err := AuthIsApiKey()
+	if err != nil {
+		return err
+	}
+	if isUsingApiKey {
+		dief("Command not available for auth with api keys")
```

Comment:
I would print something like "Authenticated with API key" and exit with success — we can augment the API key information later.